seed/analysis_pipelines/better/client.py

```diff
@@ -146,19 +146,30 @@ def is_ready(res):
             response, errors = res[0], res[1]
             if errors:
                 raise Exception('; '.join(errors))
-            return response['generation_result'] == 'COMPLETE'
 
+            if response['generation_result'] == 'COMPLETE':
+                return True
+            elif response['generation_result'] == 'FAILED':
+                raise Exception(f'BETTER failed to generate the portfolio analysis: {response}')
```

Comment:
new edge case, was getting a FAILED result b/c of bad meter data

Reply:
🔥